src/gentropy/method/window_based_clumping.py

```diff
@@ -154,22 +154,38 @@ def _prune_peak(position: NDArray[np.float64], window_size: int) -> DenseVector:
 
     @staticmethod
     def clump(
-        summary_statistics: SummaryStatistics,
+        unclumped: SummaryStatistics | StudyLocus,
         distance: int = WindowBasedClumpingStepConfig().distance,
-        gwas_significance: float = WindowBasedClumpingStepConfig().gwas_significance,
     ) -> StudyLocus:
-        """Clump significant signals from summary statistics based on window.
+        """Clump signals from summary statistics or study locus dataset based on window.
```

Comment:
Currently semi-indices only left as temporary column names. I think in this function is not relevant because it can be applied on curated associations, therefore the interpretation is different. However I'll ask Yakov to add some explanation to the summary statistics dataset. Applying the same function on summary statistics yields semi indices which might need explanation.